litmus-portal/graphql-server/pkg/projects/project_handler.go

```diff
@@ -55,8 +56,9 @@ func ProjectInitializer(ctx context.Context, projectID string, role string) erro
 	}
 
 	log.Print("Cloning https://github.com/litmuschaos/chaos-charts")
+
 	//TODO: Remove goroutine after adding hub optimisations
-	go chaoshub.AddChaosHub(context.Background(), defaultHub)
+	go chaoshub.NewService(&mongodb.MongoOperations{}).AddChaosHub(context.Background(), defaultHub)
```

Comment:
this line also creates new service instance because service logic is initialized in graph package.